spec/requests/locker_applications_spec.rb

```diff
@@ -358,5 +370,23 @@
         expect(response).to be_successful
       end
     end
+
+    describe 'PUT /toggle_archived' do
+      it 'allows an admin to archive an application' do
+        locker_application = LockerApplication.create! valid_attributes
+        put toggle_archived_locker_application_url(locker_application)
+        locker_application.reload
+        expect(response).to have_http_status(:found)
+        expect(locker_application.archived).to be(true)
+      end
+
+      it 'allows an admin to unachive an application' do
```

Comment:
@rladdusaw thank you! can you please fix the typo in the spec? unachive -> unarchive